src/Domain/Direction.hpp

```diff
@@ -10,6 +10,7 @@
 #include <iosfwd>
 #include <pup.h>
 
+#include "DataStructures/Tensor/IndexType.hpp"
```

Comment:
"Darn I don't need that"

Reply:
Below, change the constructor that takes `dimension` to `logical_dimension` and update the comment.